regulations/static/regulations/css/less/module/comment-media-queries.less

```diff
@@ -61,11 +61,27 @@ comment-media-queries.less contains media query styles for Notice and Comment
 // tablet + mobile screen sizes
 @media only screen and ( max-width: 780px ) {
 
+  .preamble-sub-head {
+
+    #content-header {
+      padding-left: 35px;
+    }
+  }
+
   .hide-mobile {
     display: none;
     padding: 0;
   }
 
+  .preamble-header {
+
+    .read-proposal,
+    .write-comment {
+      padding: 0 15px;
+      width: auto !important;
```

Comment:
Small flag around the use of `!important` -- sure there's not a better solution?